ocs_ci/utility/ibmcloud.py

```diff
@@ -203,3 +205,232 @@ def add_deployment_dependencies():
     ]
     for cr in wa_crs:
         run_cmd(f"oc apply -f {cr}")
+
+
+class IBMCloud(object):
+    """
+    Wrapper for Ibm cloud
+    """
+
+    def restart_nodes(self, nodes, timeout=900, wait=True):
+        """
+        Reboot the node on IBM Cloud.
+
+        Args:
+            nodes (list): The worker node instance
+
```

Comment:
There is some mishmash in singular and plural (`node` and `nodes`).
And also I understand the docstring in the way, that I can provide list of nodes which will be restarted, but from the logic, it looks like it uses the `nodes` variable only to get `cluster_id` and then it reboots all the existing worker nodes. If my understanding is true, please update the docstring to make it clear.

Reply:
Changed the doctoring